airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py

```diff
@@ -153,17 +153,6 @@ def get_request_body_json(  # type: ignore
             stream_state=stream_state, stream_slice=stream_slice, next_page_token=next_page_token
         )
 
-    def request_kwargs(
-        self,
-        *,
-        stream_state: Optional[StreamState] = None,
-        stream_slice: Optional[StreamSlice] = None,
-        next_page_token: Optional[Mapping[str, Any]] = None,
-    ) -> Mapping[str, Any]:
-        # todo: there are a few integrations that override the request_kwargs() method, but the use case for why kwargs over existing
-        #  constructs is a little unclear. We may revisit this, but for now lets leave it out of the DSL
-        return {}
-
     disable_retries: bool = False
     _DEFAULT_MAX_RETRY = 5
     _DEFAULT_RETRY_FACTOR = 5
```

Comment:
Not necessarily part of your PR, but I think its quite confusing to have these 3 assigned down here. Can we move these to the top of the class?

It took me a second to verify that `disable_retries` was already set on this class